Sources/sophiajs/src/sophia.ts

```diff
@@ -611,13 +615,18 @@
                     return true;
                 });
 
-                // TODO: Add error handling here.
-                // req.addEventListener('error', function (event) {
-                //     var message = { payload: { url: (event?.currentTarget as any).responseURL } };
-                //     window.webkit.messageHandlers.handler.postMessage(message);
-                // });
+                req.addEventListener('error', function (_) {
+                    self.neworkIdleMonitor.onRequestEnd();
+                    // TODO: Add error handling here.
+                    // var message = { payload: { url: (event?.currentTarget as any).responseURL } };
```

Comment:
Remove comment out code.

Reply:
will remove it, will leave the TODO, since it's actually not handled. need to make a note to cover that.